pandapower/pypower/qps_pips.py

```diff
@@ -152,8 +152,8 @@ def qps_pips(H, c, A, l, u, xmin=None, xmax=None, x0=None, opt=None):
     else:
         nx = p['H'].shape[0]
 
-    p['xmin'] = -Inf * ones(nx) if 'xmin' not in p else p['xmin']
-    p['xmax'] =  Inf * ones(nx) if 'xmax' not in p else p['xmax']
+    p['xmin'] = full(nx, -inf) * ones(nx) if 'xmin' not in p else p['xmin']
+    p['xmax'] = full(nx, inf) * ones(nx) if 'xmax' not in p else p['xmax']
```

Comment:
You can just use np.full, without multiplying with ones... but np.full is faster for larger arrays

Reply:
Thank you, i did not notice that.